### ISSUE TITLE
Mention in README needing to run "chmod" on file to get it working

### ISSUE THREAD

**Issue by timtyrrell** [dickeyxxx/tmux-spotify-info#4]
Thanks for the plugin, just missing one helpful step in the README

**Comment by ErikBoesen** [comment]
I don't think this is necessary. The script is already marked as executable. Unless you copy and paste to a new file this isn't an issue.

(I just realized how old this is, whoops.)
